### ISSUE TITLE
Building frontend throws error

### ISSUE THREAD

**Issue by manekenpix** [Seneca-CDOT/telescope#761]
**What happened**:

The frontend can't be built due to this error:
```
success open and validate gatsby-configs - 0.020s
success load plugins - 0.265s
success onPreInit - 0.005s
success delete html and css files from previous builds - 0.008s
success initialize cache - 0.006s
success copy gatsby files - 0.020s
success onPreBootstrap - 0.385s
success createSchemaCustomization - 0.003s

 ERROR #11321  PLUGIN

"gatsby-source-graphql" threw an error while running the sourceNodes lifecycle:

request to http://localhost:3000/graphql failed, reason: connect ECONNREFUSED 127.0.0.1:3000

not finished source and transform nodes - 0.082s
```

**What should have happened**:
The frontend should've been built.

**How to reproduce it (as precise as possible)**:
Run 
```
npm run clean
npm run build
```
or launch telescope using `docker-compose up`

**Environment**:

- OS: Linux
- Browser:

**Comment by Silvyre** [comment]
Could this be related to https://github.com/Seneca-CDOT/telescope/pull/747?
@c3ho

**Comment by cindyledev** [comment]
I had this error while testing #747 , I ran `npm install` and didn't have that error anymore.

**Comment by c3ho** [comment]
in `gatsby-config.js` we currently have this line

```const telescopeUrl = process.env.API_URL || `http://localhost:${process.env.PORT || 3000}`;```

This is causing an issue as the Front end can't detect any of the vars we set in the .env file. It should though, but anyways a potential change could be this

```
const telescopeUrl =
  process.env.API_URL ||
  'https://dev.telescope.cdot.systems' ||
  `http://localhost:${process.env.PORT || 3000}`;
```

We're going to check for the .env `API_URL`, if that doesn't exist we'll go for `https://dev.telescope.cdot.systems` and lastly we'll default to localhost although I doubt its ever going to use localhost

thoughts @manekenpix , @Silvyre , @cindyledev ?

**Comment by c3ho** [comment]
Fixed via reverting commit 44e329a
Related commit is - 9b019abc21c443a5998fb5089c7e5369d88a5393
